test/functional/terminal/scrollback_spec.lua

```diff
@@ -438,9 +438,29 @@ describe("'scrollback' option", function()
     command('sleep 100m')
 
     feed_data(nvim_dir.."/shell-test REP 41 line"..(iswin() and '\r' or '\n'))
-    screen:expect{any='40: line                      '}
+    if iswin() then
+      screen:expect{grid=[[
+        37: line                      |
+        38: line                      |
+        39: line                      |
+        40: line                      |
+                                      |
+        ${1: }                            |
+        {3:-- TERMINAL --}                |
+      ]]}
+    else
+      screen:expect{grid=[[
+        36: line                      |
+        37: line                      |
+        38: line                      |
+        39: line                      |
+        40: line                      |
+        {IGNORE}|
```

Comment:
is IGNORE still needed? I would think `shell-test` is predictable.

Reply:
It uses "sh" as the outer shell.
I've seen "sh-5…" so figured it might depend on the version.
No idea why there is no prompt on Windows though.

Reply:
With https://github.com/neovim/neovim/pull/11004 we could have it more explicit of course - but might still look different then when "sh" is some other shell on user systems.
So I think this is fine.